src/observer/rss.py

```diff
@@ -187,6 +188,23 @@ def article_to_rss_entry(art):
     item['author'] = [{'name': a.name, 'email': art.author_email} for a in art.authors.all()]
     item['category'] = [{'term': subject.name, 'label': subject.label} for subject in art.subjects.all()]
     item['dc:dc_date'] = utils.ymdhms(item['pubDate'])
+
+    image = {'url': "https://elifesciences.org/assets/patterns/img/patterns/organisms/elife-logo-xs.fd623d00.svg",
+             'height': "51",
+             'width': "136",
+             'type': "image/svg"}
+
+    if art.social_image_uri:
+        max_xy = 800
+        thumbnail_width, thumbnail_height = utils.thumbnail_dimensions(max_xy, art.social_image_width, art.social_image_height)
+        iiif_url = utils.iiif_thumbnail_link(art.social_image_uri, thumbnail_width, thumbnail_height)
+        image = {'url': iiif_url,
+                 'height': str(thumbnail_height),
+                 'width': str(thumbnail_width),
+                 'type': "image/jpeg"}
```

Comment:
add note explaining why this is hardcoded rather than use the mime that came with the social image